CHANGELOG.rst

```diff
@@ -16,6 +16,10 @@ User-Facing Changes
 * 📑 **UI**: Add a new dashboard for logged-in users, showing running sessions, last
   accessed projects, and own datasets
   (`#2332 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2332>`_).
+* **Sessions**: Experimental Azure Blob storage support 
```

Comment:
I'm wondering if a thorough release notes reader reads this, they'll wonder how they can use Azure blobs. Should we comment here on whether this is something a user can use right now? Or is it turned off by default? Or is this an Internal Change, and not something we should highlight in the User Facing Changes?

Reply:
This is not enabled by default and enabling it requires installing additional components outside of the Renku helm chart. Also since it is experimental I propose that we should move it to Internal Changes. I would not reccomend any Renku deployment out in the wild to use this yet.

Reply:
So this is now under Internal Changes.